inst/extdata/r_packages.R

```diff
@@ -0,0 +1,40 @@
+library(assertthat)
+library(bookdown)
+library(config)
+library(conflicted)
```

Comment:
This file is in a weird place just to make it accessible via `system.file()` in tests -- but I'm now I need that so I we may eventually move it to a more natural place.